### ISSUE TITLE
Improve article 2210.00758

### ISSUE THREAD

**Issue by jonsterling** [dginev/ar5iv#372]
**Problem details**

> Conversion to HTML had a Fatal error and exited abruptly. This document may be truncated or damaged.

It sounds like `newpxtext` and `newpxmath` are not supported.

**Desktop (please complete the following information)**
 - OS: macOS [e.g. macOS, Windows, Ubuntu]
 - Browser: Safari

**Comment by dginev** [comment]
Indeed, `newpxtext.sty`, `newpxmath.sty` and `tikz-cd.sty` need latexml support as a start.

**Comment by dginev** [comment]
ar5iv can now generate some HTML for this article, but the package support outlined above is still an open task:

> Conversion complete: 154 warnings; 61 errors; 3 undefined macros[\rrbracket, \slimits@, \llbracket]; 2 missing files[newpxtext.sty, newpxmath.sty]
